lib/engineering/analysis/footing/centric_isolated.rb

```diff
@@ -0,0 +1,48 @@
+module Engineering
+  module Analysis
+    module Footing
+      class CentricIsolated
+        ORTOGONALITIES = %i[length_x length_y].freeze
+        CUT_DIRECTION_ERROR = 'Not valid cut direction, should one of thes: :length_x :length_y'.freeze
+
+        def initialize(column:, footing:, effective_height:, load_from_column:, cut_direction:)
+          @column = column
+          @footing = footing
+          @load_from_column = load_from_column
+          @effective_height = effective_height.to_f
+
+          raise ArgumentError, CUT_DIRECTION_ERROR unless ORTOGONALITIES.any?(cut_direction)
```

Comment:
I think we can create a custom error for this, called something like `SectionDirectionError`

```ruby
module Engineering
  module Analysis
    class SectionDirectionError < StandardError
      def initialize(section_direction, valid_options)
        message = "#{section_direction} is not a valid direction, should one of #{valid_options}"

        super(message)
      end
    end
  end
end
```
We can save those errors in a separate `lib/errors/` folder. That way we can reuse this error in other analysis classes


Then this line would be something like this
```suggestion
          raise Engineering::Analysis::SectionDirectionError, cut_direction, ORTHOGONALITIES
```

And the `CUT_DIRECTION_ERROR` is not needed

Reply:
Done!